src/relay.js

```diff
@@ -171,13 +179,12 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
     return result;
   };
 
-  let resolveEdge = function (item, args = {}, source) {
-    if (!args.orderBy) {
-      args.orderBy = [defaultOrderBy];
-    }
-
+  let resolveEdge = function (item, index, cursor, args = {}, source) {
+    let startIndex = 0;
+    if (cursor) startIndex = Number(cursor.index);
+    if (startIndex !== 0) startIndex++;
```

Comment:
If there is a cursor supplied to this function, we are simply getting the index of that cursor to determine where to start numbering the next indexes. Because index in this case is going to always start at 0 we need to increment startIndex if we aren't dealing with a startIndex of 0.

Ex: we get the first three items from query, they will have a index of 0,1 and 2 in their cursor returned from the api. If we then send the cursor of the 2 to the after clause, the next 3 returned will start numbering at 2. Without the startIndex++ check the next set of results would be 2, 3 and 4 which would be wrong.